pkgs/development/libraries/openvdb/default.nix

```diff
@@ -3,19 +3,29 @@
 stdenv.mkDerivation rec
 {
   pname = "openvdb";
-  version = "9.1.0";
+  version = "10.0.1";
+
+  outputs = [ "out" "dev" ];
 
   src = fetchFromGitHub {
-    owner = "dreamworksanimation";
+    owner = "AcademySoftwareFoundation";
```

Comment:
:heavy_check_mark: https://github.com/dreamworksanimation/openvdb redirects to https://github.com/AcademySoftwareFoundation/openvdb